tests/data/outputs/event.go

```diff
@@ -0,0 +1,41 @@
+package outputs
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/falcosecurity/testing/pkg/run"
+)
+
+type Data struct {
+	Events []string `json:"events"`
+}
+
+func deserialize() string {
```

Comment:
Since we have a fairly recent version of Go, let me suggest considering go:embed https://pkg.go.dev/embed. I think it's a better approach than going through `runtime.Caller` and reinvent the wheel in this case! What do you think?

Plus, if the only point of having a JSON file is to deserialize it and encode the inner list with comma-separated concatenation, why not just encoding all strings as comma-separated in a text file right away? This would also save you the trouble of defining a `Data` struct. I see the attempt of making something reusable, but I would suggest to 1) tackle reusability later when needed and keep the scope of this PR smaller, and 2) the JSON encoding is arbitrary, so it would be less intuitive for future contributors to reproduce.

**TL;DR**: Great work! My last suggestions are:
1. Encode the file as a comma-separated text file containing the list of event names
2. Include it in the project using go:embed, and then making it available through the `NewStringFileAccessor` construct like you already do.

Reply:
>Since we have a fairly recent version of Go, let me suggest considering go:embed https://pkg.go.dev/embed. I think it's a better approach than going through runtime.Caller and reinvent the wheel in this case! What do you think?

Oh, wow I didn't know something like this existed :open_mouth: . Yes, this is great.

>1. Encode the file as a comma-separated text file containing the list of event names
>2. Include it in the project using go:embed, and then making it available through the NewStringFileAccessor construct like you already do.

Will do!!